src/repeater/repeater.js

```diff
@@ -90,47 +124,57 @@ function onFailure(err) {
 function stopAllRepeat() {
   debug('Entered repeater.stopAllRepeat');
   Object.keys(tracker).forEach((key) => {
-    if (tracker[key].stop) {
-      tracker[key].stop();
-    } else {
-      Object.keys(tracker[key]).forEach((nestedKey) => {
-        if (tracker[key][nestedKey].stop) {
-          tracker[key][nestedKey].stop();
-        }
-      });
-    }
-
-    delete tracker[key];
+    stop(key);
```

Comment:
this could be written more concisely as

`Object.keys(tracker).forEach(stop)`

which calls the function named "stop" with the arg from the loop